src/ParseQuery.js

```diff
@@ -1468,6 +1498,21 @@ class ParseQuery {
     return this;
   }
 
+  /**
+   * Sets the flag to include with response the total number of objects satisfying this query,
+   * despite limits/skip. Might be useful for pagination.
+   * Note that result of this query will be wrapped as an object with
+   *`results`: holding {ParseObject} array and `count`: integer holding total number
+   * @param {boolean} b false - disable, true - enable.
+   * @return {Parse.Query} Returns the query, so you can chain this call.
+   */
+  withCount(includeCount: boolean): ParseQuery {
+    if (typeof includeCount !== 'boolean') {
+      throw new Error('You can only set withCount to a boolean value');
+    }
+    this._count = includeCount;
```

Comment:
Can you set a default?

`withCount(includeCount: boolean = true): ParseQuery {`